src/test/java/uk/gov/companieshouse/limitedpartnershipsapi/service/LimitedPartnerServiceTest.java

```diff
@@ -20,7 +20,9 @@
 import uk.gov.companieshouse.limitedpartnershipsapi.repository.LimitedPartnerRepository;
 import uk.gov.companieshouse.limitedpartnershipsapi.utils.TransactionUtils;
 
+import java.util.ArrayList;
```

Comment:
Looks like you've removed a test from a previous commit but left unused imports and variables.

Reply:
Tidied this up.